lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -8,7 +8,7 @@ def initialize(serializer, options = {})
       end
 
       def serializable_hash(options = nil)
-        options ||= {}
+        options = super
```

Comment:
I figure this pattern with the base class is a little nicer and less error prone.